packages/graphics/harfbuzz/package.mk

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="harfbuzz"
+PKG_VERSION="2.7.3"
+PKG_SHA256="97bf10e7bc576a4e60a1cb45388669b778bb7787660d8666c4e554f021cdead9"
+PKG_LICENSE="GPL"
+PKG_SITE="http://www.freedesktop.org/wiki/Software/HarfBuzz"
+PKG_URL="https://github.com/harfbuzz/harfbuzz/releases/download/$PKG_VERSION/harfbuzz-$PKG_VERSION.tar.xz"
+PKG_DEPENDS_TARGET="toolchain cairo freetype glib icu"
+PKG_LONGDESC="HarfBuzz is an OpenType text shaping engine."
+PKG_TOOLCHAIN="meson"
+PKG_DEPENDS_CONFIG="icu"
```

Comment:
If that is a hard dependency, icu should be moved out of addon-depends as well with its PKG_BUILD_FLAGS="-sysroot" removed, and then this line can be removed.
That in return allows more cleanups with icu handling in a few packages (`git grep -w icu`).
Note that other packages might detect and use icu then as well.

Reply:
@CvH - yes disabling icu does compile. It was my first attempt at updating harfbuzz. The reason I persisted in figuring out why -Dicu=enable did not work was that icu was enabled previously. Happy either way. Let me know.

Reply:
debian has libharfbuzz and libharfbuzz-icu, where only the latter depends on libicu. libass depends on the former, so it should work build a icu less harfbuzz for the base image, and a icu enabled harfbuzz for the addon. If so desired.

Reply:
yea sounds like plan

Reply:
Done. harfbuzz renamed to harfbuzz-icu, and new harfbuzz package created.